packages/nimble-components/src/table-column/enum-text/index.ts

```diff
@@ -31,15 +31,10 @@ export class TableColumnEnumText extends mixinGroupableColumnAPI(
         >
     )
 ) {
-    protected get supportedMappingElements(): readonly (typeof Mapping)[] {
-        return [MappingText] as const;
-    }
-
     public override createValidator(): TableColumnEnumTextValidator {
-        return new TableColumnEnumTextValidator(
-            this.columnInternals,
-            this.supportedMappingElements
-        );
+        return new TableColumnEnumTextValidator(this.columnInternals, [
+            MappingText
```

Comment:
If prettier allows it, I think it would look best as:
```ts
        return new TableColumnEnumTextValidator(
            this.columnInternals,
            [MappingText]
         );
```

Reply:
It does not 👎